tests/test_integrated_channels/test_integrated_channel/test_transmitters/test_content_metadata.py

```diff
@@ -219,152 +280,87 @@ def test_transmit_update_failure(self):
         update_payload = {}
-        delete_payload = {content_id: past_transmission}
-        content_updated_mapping = {
-            content_id: {'catalog_uuid': uuid.uuid4(), 'modified': datetime.now()},
-        }
+        delete_payload = {content_id_1: content_1}
         self.delete_content_metadata_mock.side_effect = ClientError('error occurred')
         transmitter = ContentMetadataTransmitter(self.enterprise_config)
-        transmitter.transmit(create_payload, update_payload, delete_payload, content_updated_mapping)
+        transmitter.transmit(create_payload, update_payload, delete_payload)
 
         self.create_content_metadata_mock.assert_not_called()
         self.update_content_metadata_mock.assert_not_called()
         self.delete_content_metadata_mock.assert_called()
 
-        assert not ContentMetadataItemTransmission.objects.filter(
+        deleted_transmission = ContentMetadataItemTransmission.objects.get(
             enterprise_customer=self.enterprise_config.enterprise_customer,
             plugin_configuration_id=self.enterprise_config.id,
             integrated_channel_code=self.enterprise_config.channel_code(),
-            content_id=content_id,
-            channel_metadata=channel_metadata
-        ).first().deleted_at
-
-    @mark.django_db
-    @ddt.ddt
-    def test_transmitting_create_content_with_previously_deleted_record(self):
-        """
-        Test that records associated with content that was transmitted under a different catalog will be converted to
-        the new catalog
-        """
-        content_id = 'course:DemoX'
-        channel_metadata = {'update': True}
-        past_transmission = ContentMetadataItemTransmission(
-            enterprise_customer=self.enterprise_config.enterprise_customer,
-            plugin_configuration_id=self.enterprise_config.id,
-            integrated_channel_code=self.enterprise_config.channel_code(),
-            content_id=content_id,
-            channel_metadata=channel_metadata,
-            enterprise_customer_catalog_uuid=self.enterprise_catalog.uuid,
-            deleted_at=datetime.now()
+            content_id=content_id_1,
         )
-        past_transmission.save()
-        create_payload = {content_id: channel_metadata}
-        update_payload = {}
-        delete_payload = {}
-        content_updated_mapping = {
-            content_id: {'catalog_uuid': self.enterprise_catalog.uuid, 'modified': datetime.now()},
-        }
-        self.create_content_metadata_mock.return_value = (200, '{"success":"true"}')
-        transmitter = ContentMetadataTransmitter(self.enterprise_config)
-        assert past_transmission.deleted_at
-        transmitter.transmit(create_payload, update_payload, delete_payload, content_updated_mapping)
-        past_transmission.refresh_from_db()
-        assert not past_transmission.deleted_at
 
-    @mark.django_db
-    @ddt.ddt
-    def test_transmitting_same_content_over_different_catalogs(self):
```

Comment:
the catalog behavior needs to be reworked - it has little value since courses can exist in multiple, overlapping catalogs